examples/install_vars.yaml

```diff
@@ -9,5 +9,6 @@ workdir: ~/ocp4-workdir
 storage_type: nfs
 log_level: info
 release_image_override: ""
+local_registry: ""
```

Comment:
Why get resgistry value from user?
This can be generated on fly using cluster id and domain.

Reply:
It also needs local repo value with it. should we create a seperate var for that?